### ISSUE TITLE
Not able to render custom cursor

### ISSUE THREAD

**Issue by Take-A-Byte** [MicrosoftEdge/WebView2Feedback#3388]
**Description**

If we try to use a custom cursor inside a UWP app, we get a default cursor instead of a specified cursor. It does not even render the fallback cursor.

**Version**
SDK: 1.0.1661.34
Framework: UWP
OS: Win11

**Repro Steps**
1. Load local index.html with the following content in a UWP app inside WebView2
``` HTML
<!DOCTYPE html>
<html lang="en">
<head>
    <style>
        html, body {
            width: 100%;
            height: 100%;
        }

        #root {
            width: 100%;
            height: 100%;
            background: aqua;
            cursor: url("circle.ico"), crosshair;
        }
    </style>
</head>
<body>
<div id="root"></div>
</html>
```

**Expected Output**
Cursor is rendered as `circle.ico` or crosshair

**Actual Output**
Default cursor is rendered

**Additional context**
If we use `cursor: crosshair;` instead, the crosshair is rendered

**Comment by david-risney** [comment]
This is an existing issue for WebView2 in UWP apps: #1925. I'm going to make sure we get this listed in our docs for now.

**Comment by david-risney** [comment]
I've added a section to this doc https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2#custom-cursors about custom cursors not working. That doc covers various things that are currently not supported in WebView2 in UWP and that content should be moved in the near future into its own separate doc (its sort of hard to find right now in the Getting Started doc)

**Comment by Take-A-Byte** [comment]
This is helpful! Thanks 🙇

**Comment by Take-A-Byte** [comment]
@david-risney I just noticed that `cursor: none` also does not work. Could you please verify this and add it to the [documentation](https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2#custom-cursors)?

**Comment by david-risney** [comment]
Yes, thanks. Looks like some of the other built-in cursor kinds don't work as well. Like help, wait, crosshair work, but progress, grab, don't. I'll update that, thanks.

**Comment by Take-A-Byte** [comment]
Thanks 🙇

**Comment by orosbogdan** [comment]
Any updates on this issue, facing the same in Blazor Hybrid with .NET 8.

**Comment by 0exception** [comment]
is there any way to use custom cursor in webview2?

**Comment by elias-ik** [comment]
Some of the resize cursors don't work either, for example ew-resize doesn't work, while col-resize does work.

**Comment by ChaseIngersol** [comment]
Any updates? Still running into this today...

**Comment by wood4099** [comment]
However, there are still issues
